### PR TITLE
Merging the r1.10 branch back to master.

### PR DESCRIPTION
**Pull request by av8ramit**

### CODE REVIEW

File: tensorflow/docs_src/install/install_mac.md [hunk 1]
```diff
@@ -350,7 +350,11 @@ Take the following steps to install TensorFlow in an Anaconda environment:
      TensorFlow for Python 2.7:
 
      <pre> (<i>targetDirectory</i>)$ <b>pip install --ignore-installed --upgrade \
+<<<<<<< HEAD
```
Comment (gunan) [inline tensorflow/docs_src/install/install_mac.md]: resolve conflicts?

File: tensorflow/docs_src/install/install_sources.md [hunk 2]
```diff
@@ -282,6 +282,7 @@ Please specify the NCCL version you want to use. If NCLL 2.2 is not installed, t
 
 Please specify a list of comma-separated Cuda compute capabilities you want to build with.
 You can find the compute capability of your device at: https://developer.nvidia.com/cuda-gpus.
+<<<<<<< HEAD
```
Comment (gunan) [inline tensorflow/docs_src/install/install_sources.md]: also resolve conflicts

Comment (av8ramit) [reply tensorflow/docs_src/install/install_sources.md]: Done.

File: tensorflow/docs_src/install/install_mac.md (hunk 1 above)
Comment (av8ramit) [reply tensorflow/docs_src/install/install_mac.md]: Done. Must have missed it.

**Review by gunan — APPROVED** [review]

**Pull request merged**